coil-base/build.gradle.kts

```diff
@@ -14,20 +12,7 @@ plugins {
     id("org.jetbrains.dokka")
 }
 
-android {
-    compileSdkVersion(project.compileSdk)
-    defaultConfig {
-        minSdkVersion(project.minSdk)
-        targetSdkVersion(project.targetSdk)
-        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
-    }
-    libraryVariants.all {
-        generateBuildConfigProvider?.configure { enabled = false }
-    }
-    testOptions {
-        unitTests.isIncludeAndroidResources = true
-    }
-}
+setupBase()
```

Comment:
This looks good, but could we rename `setupBase` -> `setupLibraryModule` to be more explicit?

Reply:
Renamed.